src/FluentlyHttpClient/FluentHttpRequestBuilderExtensions.cs

```diff
@@ -91,5 +135,22 @@ public static class FluentHttpRequestBuilderExtensions
 			var response = await builder.ReturnAsResponse().ConfigureAwait(false);
 			return await response.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
 		}
+
+
+		/// <summary>
+		/// Send request and return as Graph QL response
+		/// </summary>
+		/// <typeparam name="T"></typeparam>
+		/// <returns>Returns content within Graphql data response object</returns>
+		public static async Task<FluentHttpResponse<GqlResponse<T>>> ReturnAsGqlResponse<T>(this FluentHttpRequestBuilder builder)
+		{
+			var response = await builder.ReturnAsResponse().ConfigureAwait(false);
```

Comment:
you can use `ReturnAsResponse<T>` directly e.g. `=> builder.ReturnAsResponse<GqlResponse<T>>();` (and remove all of it)